addRestaurant.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
+  jQuery.validator.setDefaults({
+    debug: true,
+    success: 'valid',
+  });
+
+  $.each(categories, function(index, category) {
+    $('.categorySelect').append(
+      `<option value="${category.id}">${category.name}</option>`
+    );
+  });
+
+  $('.addRestaurantButtonModal').on('click', function() {
```

Comment:
if button in modal has type="submit" and is a descendant of the form, this action is not needed.